icu4j/build.properties

```diff
@@ -4,9 +4,9 @@
 #* Copyright (C) 2009-2016, International Business Machines Corporation and    *
 #* others. All Rights Reserved.                                                *
 #*******************************************************************************
-api.report.version = 67
-api.report.prev.version = 66
-release.file.ver = 67_1
-api.doc.version = 67.1
-maven.pom.ver = 67.1
+api.report.version = 68
+api.report.prev.version = 67
+release.file.ver = 68_1
+api.doc.version = 68.1
+maven.pom.ver = 68.1
```

Comment:
We'll want this to be the following for the Release Candidate (RC):
```
release.file.ver = 68rc
api.doc.version = 68 Release Candidate
maven.pom.ver = 68.1-SNAPSHOT
```

Reply:
Oh I didn't know that :D Thanks Daniel